spec/checkPullRequestReviewSpec.js

```diff
@@ -804,10 +807,11 @@ describe('Pull Request Review Module', () => {
                 },
               }),
           };
-          github.orgs = {
-            checkMembership: jasmine.createSpy('checkMembership')
+          github.repos = {
+            getCollaboratorPermissionLevel: jasmine
+            .createSpy('getCollaboratorPermissionLevel')
               .and.resolveTo({
-                status: 204,
+                status: 200,
```

Comment:
This should resolve to an object with the following structure `{data:{status:<status>, permission<permission>}}`

Reply:
Done